nixos/modules/services/networking/jotta-cli.nix

```
@@ -0,0 +1,43 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.user.services.jotta-cli;
+in {
+  options = {
+    user.services.jotta-cli = {
+
+      enable = mkEnableOption "Jottacloud Command-line Tool";
+
+      options = mkOption {
+        default = [ "stdoutlog" "datadir" "%h/.jottad/" ];
+        example = [ ];
+        type = with types; listOf str;
+        description = mdDoc "Command-line options passed to jottad.";
```

```suggestion
        description = "Command-line options passed to jottad.";
```
that got recently dropped

---

Fixed

---

@SuperSandro2000 Merged this in manually, could you mark the requested changes as fixed?